### DISCUSSION TITLE
GitHub Student status verified but benefits still not activated.

### DISCUSSION THREAD

**Discussion by satyam-capcons** [community/community#56029] [GitHub Education]
### Hi everyone,

Hi everyone, 
Even though my student status has been verified, the benefits such as students developers pack and stuff are not yet activated. Any reasons for the delay?

**Comment by aviiciii** [comment]
Hey, try going to this url https://education.github.com/pack and signing in to your verified accound, you should be able to see the student developer pack activated.

**Comment by satyam-capcons** [comment]
It's still not showing as activated.

**Reply by Blira** [reply]
How long did it take to activate for you? I'm waiting for about 2 days now

**Comment by satyam-capcons** [comment]
You should receive in 1-2 days more

**Comment by samus-aran** [comment]
Hey @satyam-capcons  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away if successful. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
